packages/react/src/Heading/Heading.test.tsx

```diff
@@ -15,91 +15,54 @@ describe('Heading', () => {
     expect(heading).toBeVisible()
   })
 
-  it('renders a default heading at heading level 1', () => {
-    render(<Heading>Breaking news</Heading>)
-
-    const heading = screen.getByRole('heading', {
-      name: 'Breaking news',
-      level: 1,
-    })
-
-    expect(heading).toBeInTheDocument()
-  })
-
-  it('renders a heading at the specified level', () => {
+  it('renders the other allowed levels correctly', () => {
     render(
       <>
-        <Heading level={1}>H1</Heading>
-        <Heading level={2}>H2</Heading>
-        <Heading level={3}>H3</Heading>
-        <Heading level={4}>H4</Heading>
+        <Heading level={2}>Heading 2</Heading>
+        <Heading level={3}>Heading 3</Heading>
+        <Heading level={4}>Heading 4</Heading>
       </>,
     )
 
-    const h1 = screen.getByRole('heading', {
-      name: 'H1',
-      level: 1,
-    })
-    const h2 = screen.getByRole('heading', {
-      name: 'H2',
-      level: 2,
-    })
-    const h3 = screen.getByRole('heading', {
-      name: 'H3',
-      level: 3,
-    })
-    const h4 = screen.getByRole('heading', {
-      name: 'H4',
-      level: 4,
-    })
+    const h2 = screen.getByRole('heading', { level: 2 })
```

Comment:
Using a name here is kind of useful I think. If `level={3}` sets a level 4 heading, and vice versa for example, this current test will still pass, even though it shouldn't.

Reply:
I don’t understand. How can `level={3}` set a level 4 heading?

Reply:
If we make a mistake in the future 😁 Tests should catch mistakes, the current test would not catch that mistake.

Reply:
I still don’t get how it helps, but I’ve restored the names.